src/lib.rs

```diff
@@ -22,16 +22,23 @@ pub struct FmtDiff {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-struct AtomEdit {
-    delete: TextRange,
-    insert: SmolStr,
+pub struct AtomEdit {
+    pub delete: TextRange,
+    pub insert: SmolStr,
 }
 
 impl FmtDiff {
+    /// Get the diff of deletes and inserts
+    pub fn diff(&self) -> &[AtomEdit] {
```

Comment:
Let's change this to 

```
pub fn text_diff(&self) -> Vec<AtomEdit>
```

By exposing `&[]` we expose the fact that we store an `Vec` inside, but it is reasonable if this moves to tree-based diff in the future